ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -172,6 +194,12 @@ class ScanQrCodeScreen extends React.Component<Props, State> {
           .catch(() => {
             this.onInvalidQrCode();
           });
+      } else {
+        this.settingsAlert(
+          I18n.t("wallet.QRtoPay.settingsAlert.title"),
+          I18n.t("wallet.QRtoPay.settingsAlert.message"),
+          openAppSettings
```

Comment:
On this alert (the second one below), we could place the button that allows the user to proceed with the selected  functionality on right as the first alert
<img width="189" alt="image" src="https://user-images.githubusercontent.com/38431762/72903524-a63c1700-3d2d-11ea-8104-090cb65183c3.png">

Reply:
@valentinacala what do you think about it

**Android**
<img width="474" alt="Schermata 2020-01-22 alle 16 44 25" src="https://user-images.githubusercontent.com/7993551/72908910-79403200-3d36-11ea-8e60-cef88855fe4f.png">

**iOS**
<img width="252" alt="Schermata 2020-01-22 alle 16 44 16" src="https://user-images.githubusercontent.com/7993551/72908925-7f361300-3d36-11ea-84c3-953f3785192e.png">

Reply:
I'm checking on the app and we have different messages for each alert related to permissions. 

Here a summary (for iOS) with changes to unify the messages displayed by the user. @Edigio , @Undermaken  do you think?
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/38431762/72917755-b6132580-3d44-11ea-8a15-f5d0c871d6c1.png">